.jpb/persistence-units.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What are these two xml files needed for?

Reply:
@GuentherJulian  Done with the investigation. persistance-units.xml is the file that is used to connect the entities created later on during tests to the database. They define connection attributes of the local db. 

This file isn't complete because the configuration attributes weren't defined in the ticket hence the configuration would be done before the testing phase. 

Long story short, we can remove this file but it's going to be created once again anyway.